tests/test_actors.py

```diff
@@ -68,17 +68,18 @@ def _callback_timer(self, task):
         time.sleep(0.5)  # force one token to "take" 0.5 s
         task['exit_code'] = 0
 
-    def test_max_time(self):
+    def test_max_total_time(self):
         """
         Test to stop running when the max time is about to be reached.
         """
         self.count = 0
-        self.max_time = 0.5  # one token takes 0.5, so it quits after 1 token
-        self.avg_time_fac = 0.5
-        self.test_number = 1
+        max_time = 1.
         runner = MockRunWithStop(self._callback_timer)
-        runner.run(max_time=self.max_time, avg_time_factor=self.avg_time_fac)
-        self.assertEqual(self.count, self.test_number)
+        start = time.time()
+        runner.run(max_total_time=max_time)
+        end = time.time()
+        exec_time = end-start
+        self.assertAlmostEqual(max_time, exec_time, 1)
```

Comment:
How much does this assert round up? We dont want a 1 ms process to be rounded up to 1 second if it makes the test pass. The test should be in the same order of magnitude in terms of timing.

Reply:
It rounds up to 1 decimal. Since it concerns max_total_time of a pilot job, I don't think more accuracy is needed. I would think in the order of seconds is already sufficient, but you don't want the test to take up too much time either.